tsconfig.cjs.json

```diff
@@ -3,6 +3,6 @@
   "compilerOptions": {
     "target": "es2015",
     "module": "commonjs",
-    "outDir": "lib/commonjs"
+    "outDir": "dist/commonjs"
```

Comment:
is the point of this change so that we can get rid of the setup-lib-types script?

Reply:
this is so we can just have package.json files in `lib/commonjs` and `lib/esm` that point to the correct index.js file for each. otherwise, the package.json was copied in from the top-level one and the `main` and `module` paths would be incorrect, causing `@yext/answers-core/lib/commonjs` and `@yext/answers-core/lib/esm` imports to fail. removing the script is just a side effect as it's no longer necessary